src/views/CreateMemorandum.vue

```diff
@@ -18,3 +18,7 @@ import { UploadMemorandum } from '@/components/CreateMemorandum'
 })
 export default class CreateMemorandum extends Vue {}
```

Comment:
WIP i imagine? :)

Reply:
no, this is done.  it's just a container component

Reply:
oh! Ok right on